alinka/widget/containers/main_body/content/application/application_tabs/school_tab/__init__.py

```diff
@@ -84,61 +84,106 @@ def __init__(self, parent: QWidget):
 
         self.model = QStandardItemModel()
         self.model.setHorizontalHeaderLabels(["Nazwa Szkoły", "Typ Szkoły", "Adres", "Miejscowość"])
+        self.model.itemChanged.connect(self.on_checkbox_changed)
 
         self.table_view = QTableView(self)
         self.table_view.setModel(self.model)
         self.table_view.setEditTriggers(QTableView.NoEditTriggers)
-        self.table_view.setSelectionBehavior(QTableView.SelectRows)
-        self.table_view.setSelectionMode(QTableView.SingleSelection)
+        self.table_view.setSelectionMode(QTableView.NoSelection)
         self.table_view.resizeColumnsToContents()
-        self.table_view.selectionModel().selectionChanged.connect(self.selection_changed)
 
         header = self.table_view.horizontalHeader()
         header.setSectionResizeMode(QHeaderView.Stretch)
-        self.populate_school_list()
 
         self.handle_school_frame = HandleSchoolFrame(self)
         self.handle_school_frame.school_list_changed.connect(self.populate_school_list)
 
+        self.populate_school_list()
+
         layout.addWidget(self.school_type)
         layout.addWidget(self.table_view)
         layout.addWidget(self.handle_school_frame)
 
+    def on_checkbox_changed(self, item):
```

Comment:
This behaviour (checkboxes in table + only one is selected) will be applied not only to School Tab, but also Team (Zespół) Tab, right? If that’s the case, that could be extracted as a util function as it’s going to be reused.

Reply:
It works differently on "Zespół". There you can select multiple team members. Here only one school. Let me take a look if we can implement it differently.